TextToSpeech/doc/TextToSpeechPlugin.md

```diff
@@ -235,6 +240,7 @@ TextToSpeech interface events:
 | Event | Description |
 | :-------- | :-------- |
 | [onspeechstart](#event.onspeechstart) | Notifies when speech starts |
+| [onSpeechComplete](#event.onSpeechComplete) | Notifies when speech completes |
 | [onspeechcancel](#event.onspeechcancel) | Notifies when speech is cancelled |
 | [onnetworkerror](#event.onnetworkerror) | Notifies when network error is occurred |
 | [onplaybackerror](#event.onplaybackerror) | Notifies when playback error |
```

Comment:
1) "onspeechinterrupted" should be exposed / documented?
2) "onSpeechComplete" case is not matching with other events

Reply:
Updated the onSpeechComplete event properly.
Currently we planned to share minimal events like onspeechstart, onspeechcomplete, onspeechcancel, onspeechplaybackerror,onspeechnetworkerror. 
We can add more events when the requirement expands.